src/components/Authentication/LoginPage.js

```diff
@@ -92,75 +122,317 @@ const LoginPage = () => {
+                </Typography>
+                </div>
+                <form className="Login">
+                    <Grid container>
+                        <Grid item md={6}></Grid>
+                        <Grid item md={6}>
+                            <Typography className="welcomeText">
+                                Welcome to Summit
+                            </Typography>
+                            <TextField error={hasError || email === ""} fullWidth
+                                inputProps={{ "data-cy": "emailField" }}
+                                margin="normal"
+                                onChange={handleTextInput(setEmail)}
+                                value={email}
+                                placeholder="E-Mail"
+                                variant="outlined"
+                                className="TextField"
+                                helperText={hasError ? "Sorry, we couldn't find a user for that email." : " "}
+                                fullWidth="true"
+                                InputProps={{
+                                    startAdornment: (
+                                        <InputAdornment position="start">
+                                            <EmailOutlinedIcon style={{ color: "grey" }} />
+                                        </InputAdornment>
+                                    ),
+                                }} />
+                            <Grid container item className="buttonContainer">
+                                <Grid item md={2} />
+                                <Grid item md={4}>
+                                <Button className="createAccountButton"
                             component={Link} to={{
-                                "pathname": "/newaccount",
+                                "pathname": "/new/parent",
                                 "state": {
                                     email,
                                     password,
                                 },
                             }} variant="outlined">
-                            New Account
+                            CREATE ACCOUNT
                         </Button>
+                                </Grid>
+                                <Grid item md={4}>
+                                    <Button className="signInButton" onClick={() => handleCheck()}>
+                                        SIGN IN
+                                </Button>
+                                </Grid>
+                                <Grid item md={2} />
+                            </Grid>
+                        </Grid>
                     </Grid>
-                </Grid>
-            </form>
-            {hasError && (
-                <Typography color="error" data-cy="errorMessage">
-                    Invalid credentials
+                </form>
+            </div>)
+    }
+
+    const renderOtherLogins = () => {
```

Comment:
Each login page should be its own component. Otherwise your file gets too long/difficult to maintain. 

Instead of using a switch statement do this:
```
return {
    "Parent": <ParentLogin/>,
    "Student":<StudentLogin/>,
    "Instructor":<InstructorLogin/>,
    "Receptionist":<ReceptionistLogin/>
}[userType]
```
What you're doing here is you're creating an object and immediately calling it. this is much easier to maintain than having a long switch statement.

Reply:
When you separate each Login to its own file, you'll want to make each of these logins "dumb" components, meaning they don't process/edit data. Here's a [tutorial that you'll need to do this](https://webomnizz.com/change-parent-component-state-from-child-using-hooks-in-react/#:~:text=Change%20Parent%20Component%20State%20from%20Child%20using%20hooks%20in%20React,-Posted%20on%20September&text=React%20hooks%20are%20introduced%20in,callback%20function%20to%20the%20parent.) the idea is that you want your child (specific login components) to update data in your parent component (LoginPage)

Reply:
I'm not even sure if you want to make these into separate components... can you tell me what's the unique code between all of these components? I'm a bit confused since it looks like the code is nearly identical

Reply:
I agree it is identical and probably could be fit into one component, but I've separated them out to make it more flexible in case they want to change parts of the different user logins in the future.

Reply:
Took a look at the dummy page and tried to apply it to my code. It renders everything properly, but I'm struggling to figure out how to pass the more complex callback functions into my child component.

Reply:
what functions are you trying to pass?